### PR TITLE
Add notice to claiming docs about Space admins

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

The team discovered that a Space's claiming token should only be accessible to that Space's administrators. This PR updates the claiming documentation to say explicitly that only an admin can view the claiming screen and get the token+script.

I also changed some of the wording to align it with our Cloud standards, e.g. workspace -> Space.

cc @rduffield @manos-saratsis 

##### Component Name

claim/

### CODE REVIEW

File: claim/README.md [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
-common invitation mechanism, which begins with the administrators of the workspace creating a **claiming-token**. They,
-or other users is their organization, can then use the claiming-token to add an agent to their workspace.
+Agent claiming is part of the onboarding process when creating a Space in Netdata Cloud. Each Space gets its own
+common invitation mechanism, which begins with the administrators of the Scape creating a **claiming token**. They,
+or other users is their organization, can then use the claiming token to add an node to their Space.
 
-To claim a Netdata agent, you first send a claiming request to Netdata Cloud (from the agent node). Once the
-Netdata Cloud validates the claiming request of the agent (based on the claiming token), and returns a successful
-result, the node is considered claimed.
+To claim a node, you first send a claiming request to Netdata Cloud (from the agent node). Once the Netdata Cloud
```
Comment (rduffield) [inline claim/README.md:15]: I don't think the "the" is necessary.

File: claim/README.md [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
 -->
 
 # Agent claiming
 
-Agent claiming is part of the onboarding process when creating a workspace in Netdata Cloud. Each workspace gets its own
-common invitation mechanism, which begins with the administrators of the workspace creating a **claiming-token**. They,
-or other users is their organization, can then use the claiming-token to add an agent to their workspace.
+Agent claiming is part of the onboarding process when creating a Space in Netdata Cloud. Each Space gets its own
+common invitation mechanism, which begins with the administrators of the Scape creating a **claiming token**. They,
+or other users is their organization, can then use the claiming token to add an node to their Space.
```
[the remark this answers is not in the document]
Comment (amoss) [reply claim/README.md:13]: Also scape->space

File: claim/README.md [hunk 3]
```diff
@@ -51,7 +55,8 @@ This reloads the agent claiming state from disk.
 
 ## Netdata agent command line
 
-The user can trigger agent claiming by calling the `netdata` service binary with the additional command line parameters:
+A Space's administrator can also trigger agent claiming by calling the `netdata` service binary with the additional
+command line parameters:
```
Comment (amoss) [inline claim/README.md:59]: We should explain that this only works if netdata is it already running, I.e as an initialization or installation step.

File: claim/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ilyam8) [reply claim/README.md:13]: > is their organization

i guess should be **in**

[the remark this answers is not in the document]
Comment (joelhans) [reply claim/README.md]: Fixed all these (and the others as well).

**Comment by joelhans** [comment]
Closing in favor of #8724

**Pull request closed without merging**